doc/Administration-guide/Tenants-and-applications-management/index.md

```diff
@@ -10,83 +10,75 @@ sort_idx: 40
 * TOC
 {:toc}
 
+To follow this guide, you need to have your [Kaa platform installed]({{root_url}}Administration-guide/System-installation/) and the [Administration UI]({{root_url}}Glossary/#administration-ui) available from the web.
 
-This guide assumes that Kaa has already been installed and Admin UI is available from the web. If it's not the case, look at the [Installation guide]({{root_url}}Administration-guide/System-installation/) for more info.
+## Kaa users
 
-# First registration
+The first time you log in to Administration UI, it is required that you register as a Kaa administrator user.
 
-The first time you log in to Kaa Admin UI, it is required that you register as a [Kaa admin](#kaa-admin) user.
+![](attach/image2015-5-28%2017-7-0.png)
```

Comment:
replace screenshot with this one
http://storage7.static.itmages.com/i/17/0127/h_1485525990_7753072_506e1c85ac.png